Installation/nnAudio/utils.py

```diff
@@ -10,7 +10,7 @@
 import math
 from scipy.signal import get_window
 from scipy import signal
-from scipy import fft
+from scipy.fft import fft
```

Comment:
`from scipy.fftpack import fft` might be even a better choice.

Reply:
Done!